### PR TITLE
Fetch by name

### PR DESCRIPTION
**Pull request by ShireenAzad**

### CODE REVIEW

File: src/main/java/com/everestengineering/employeeportalapplication/entities/Address.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @Column(name = "number", nullable = false)
+    private Long number;
+
+    @ManyToOne(optional = false)
+    @JoinColumn(name = "id", nullable = false)
+    private Employee id;
+
+    @Lob
```
Comment (sivaprasad-ee) [inline src/main/java/com/everestengineering/employeeportalapplication/entities/Address.java:17]: Why all the columns are @Lob?

File: src/main/java/com/everestengineering/employeeportalapplication/repositories/EmployeeRepository.java [hunk 2]
```diff
@@ -1,48 +1,17 @@
...
 import org.springframework.stereotype.Repository;
 
-import java.util.ArrayList;
-import java.util.HashMap;
+import javax.transaction.Transactional;
 import java.util.List;
-import java.util.Map;
-import java.util.concurrent.atomic.AtomicLong;
 
+@Transactional
```
Comment (sivaprasad-ee) [inline src/main/java/com/everestengineering/employeeportalapplication/repositories/EmployeeRepository.java:11]: `@Transactional` should be at Service level.

Comment (ShireenAzad) [reply src/main/java/com/everestengineering/employeeportalapplication/repositories/EmployeeRepository.java:11]: Ok Siva.

File: src/main/java/com/everestengineering/employeeportalapplication/controllers/EmployeeController.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/employees")
+public class EmployeeController {
+    private final EmployeeService employeeService;
+
+    @GetMapping("/{firstName}/{lastName}")
```
Comment (sivaprasad-ee) [inline src/main/java/com/everestengineering/employeeportalapplication/controllers/EmployeeController.java:23]: Update this match `/api/employees/search?query=keywork` format

**Review by sivaprasad-ee — APPROVED** [review]

**Pull request merged**